packages/components/app/styles/components/dropdown.scss

```diff
@@ -110,36 +111,41 @@ $hds-dropdown-list-item-border-radius: 5px;
   }
 }
 
-
-// MORE
+// TOGGLE/MORE
 
 // these values later may come from the design tokens
 $size-props: (
-  "small": (
-    "size": 28px
+  'small': (
+    'size': 28px,
+  ),
+  'medium': (
+    'size': $hds-dropdown-toggle-base-height,
   ),
-  "medium": (
-    "size": $hds-dropdown-toggle-base-height
-  )
 );
 
-.hds-dropdown-toggle--more {
-  @each $size in $hds-dropdown-toggle-more-sizes {
-    &.hds-dropdown-toggle--size-#{$size} {
-      height: map-get($size-props, $size, "size");
-      width: map-get($size-props, $size, "size");
+.hds-dropdown-toggle-overflow {
+  @each $size in $hds-dropdown-toggle-overflow-sizes {
+    &.hds-dropdown-toggle-overflow--size-#{$size} {
+      height: map-get($size-props, $size, 'size');
+      width: map-get($size-props, $size, 'size');
     }
   }
+
+  &:active,
+  &.is-active {
+    background-color: var(--token-color-surface-interactive-active);
+    border-color: var(--token-color-border-strong);
+  }
```

Comment:
[chore] I suspect you don't need these, it's already declared in the top part of the file (lines 96-100). can you check?

Reply:
removed